### PR TITLE
feat: send event when ora submission is created

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PR adds the dispatch of an event when an ORA submission is created.

### How to Test
1. Install this repo with this branch in your environment.
2. In a unit of the course create a new ora assessment, maybe Staff Assessment Only.
3. In the LMS send a submission.

### CODE REVIEW

File: openassessment/xblock/ui_mixins/legacy/handlers_mixin.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 import logging
 
+from platform_plugin_turnitin.events.data import ORASubmissionData
+from platform_plugin_turnitin.events.signals import ORA_SUBMISSION_CREATED
```
Comment (mariajgrimaldi) [inline openassessment/xblock/ui_mixins/legacy/handlers_mixin.py]: This is a non-blocking comment: I know we've discussed adding the plugin directly here, but what do you think about extracting the event/data into our openedx-events fork? Then, it'd be easier to contribute to the community. The changes would be moving the events to the fork and installing the opened-events fork here instead of the PyPI release. We can do it if it's not too much work.

File: openassessment/xblock/ui_mixins/legacy/handlers_mixin.py [hunk 2]
```diff
@@ -61,6 +63,26 @@ class LegacyHandlersMixin:
...
 
+    @staticmethod
+    def send_ora_submission_created_event(submission: dict) -> None:
+        """
+        Send an event when a submission is created
+
+        Args:
+            submission (dict): The submission data
+        """
+        from openassessment.xblock.openassessmentblock import OpenAssessmentBlock
```
Comment (mariajgrimaldi) [inline openassessment/xblock/ui_mixins/legacy/handlers_mixin.py:75]: why is this necessary here instead of importing on top of the file?

**Review by mariajgrimaldi — APPROVED** [review]
Just a few comments to address. Thanks!

File: openassessment/xblock/ui_mixins/legacy/handlers_mixin.py (hunk 1 above)
Comment (BryanttV) [reply openassessment/xblock/ui_mixins/legacy/handlers_mixin.py]: It makes sense what you propose. I'll make the corresponding changes, it won't take much work.

File: openassessment/xblock/ui_mixins/legacy/handlers_mixin.py (hunk 2 above)
Comment (BryanttV) [reply openassessment/xblock/ui_mixins/legacy/handlers_mixin.py:75]: This is to avoid circular import error

**Comment by BryanttV** [comment]
These changes are already in master 💯
